cmake error

i use the cmakeList as follow
```
cmake_minimum_required(VERSION 3.16)
project(example)
list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
include(libigl)
igl_include(glfw)
igl_include(tetgen)
file(GLOB SRC_FILES *.cpp)
add_executable(${PROJECT_NAME} ${SRC_FILES})
target_link_libraries(${PROJECT_NAME} PUBLIC igl::glfw  igl_copyleft::tetgen)
```
then use 
`cmake -DFETCHCONTENT_SOURCE_DIR_LIBIGL=/home/chenpeng/Documents/libigl/libigl ..`
 to build project, I get error message 
```
CMake Error at /home/chenpeng/Documents/libigl/libigl/cmake/igl/igl_include.cmake:32 (include):
  include could not find requested file:

    /home/chenpeng/Documents/libigl/libigl/cmake/igl/modules//tetgen.cmake
Call Stack (most recent call first):
  /home/chenpeng/Documents/libigl/libigl/cmake/igl/igl_include.cmake:54 (_igl_include_full)
  CMakeLists.txt:11 (igl_include)
```

how to fix it ?

--- a/docs/example-project.md
+++ b/docs/example-project.md
@@ -61,3 +61,20 @@ From within the `build` directory just issue:
 A glfw app should launch displaying a 3D cube.
 ![Libigl example project: https://github.com/libigl/libigl-example-project](images/libigl-example-project.png)
 
+## Enabling optional modules via cmake
+
+The example project already requires the `glfw` module. This shows up in the `CMakeLists.txt` in two places:
+
+```
+igl_include(glfw)
+…
+target_link_libraries(${PROJECT_NAME} PUBLIC igl::glfw)
+```
+
+Suppose you'd like to add another module, for example, so that you can include `#include <igl/copyleft/tetgen/tetrahedralize.h>` then you'd change these to:
+
+```
+igl_include(copyleft tetgen)
+…
+target_link_libraries(${PROJECT_NAME} PUBLIC igl::glfw igl_copyleft::tetgen)
+```

